spec/linter-codeclimate-spec.js

```diff
@@ -35,4 +37,21 @@ describe('The codeclimate provider for Linter', () => {
       ),
     ),
   );
+
+  it('runs all configured engines', () =>
```

Comment:
Since there no longer are any hardcoded engines in this PR, I'd say just to remove this test since it's doing the same thing as the existing one.